core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -401,6 +401,44 @@ paths:
                 $ref: '#/components/schemas/ErrorReport'
         500:
           $ref: '#/components/responses/RawlsInternalError'
+    patch:
```

Comment:
Is this a thing that needs an equivalent passthrough in orch?

Reply:
I'm not sure- I don't think anything will _need_ this to be added to Orch unless AoU wants to use this, but I'm also not sure what our default procedure is these days. Do we add all new Rawls endpoints to Orch by default?